eks-anywhere-vsphere/Addons/Core/storage-driver/vmware-csi/vmware-csi-kustomization.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: vmware-csi-namespace-kustomization
+  namespace: flux-system
+  labels:
+    app: vmware-csi-driver
+spec:
+  interval: 1m0s
+  path: ./eks-anywhere-vsphere/Addons/Core/storage-driver/vmware-csi/vmware-csi-namespace
```

Comment:
You need 3 Kustomizations for Ordering?

Reply:
I have tested it end to end and confirmed it works, I can test again if we'd like screenshots. 

3 Kustomizations to enforce ordering was to ensure the driver would deploy after the secrets because immutability of the deployment was causing it to fail if the secret took longer than the driver. I did enable restarts however after creating this  order enforcement so I'll test to see if we can simplify it with that enabled when I re-test for screenshots

Reply:
Yes lets get some screenshots to PR. Its lot changes so we want to make sure we document the outputs.